datacite/schemas/datacite-v4.1.json

```diff
@@ -0,0 +1,700 @@
+            "required": [
+              "pointLongitude",
+              "pointLatitude"
+            ]
+          },
+          "geoLocationBox": {
+            "description": "The spatial limits of a box.",
+            "type": "object",
+            "properties": {
+              "westBoundLongitude": {
+                "description": "Western longitudinal dimension of box.",
+                "type": "number"
+              },
+              "eastBoundLongitude": {
+                "description": "Eastern longitudinal dimension of box.",
+                "type": "number"
+              },
+              "southBoundLatitude": {
+                "description": "Southern latitudinal dimension of box.",
+                "type": "number"
+              },
+              "northBoundLatitude": {
+                "description": "Northern latitudinal dimension of box.",
+                "type": "number"
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "westBoundLongitude",
+              "eastBoundLongitude",
+              "southBoundLatitude",
+              "northBoundLatitude"
+            ]
+          },
+          "geoLocationPlace": {
+            "description": "Spatial region or named place where the data was gathered or about which the data is focused.",
+            "type": "string"
+          },
+          "geoLocationPolygon": {
+            "description": "WARNING: This field has been superseded by 'geoLocationPolygons'.",
+            "type": "object",
+            "properties": {
+              "polygonPoints": {
+                "description": "A point location in a polygon. WARNING: Last point will be automatically added.",
+                "type": "array",
+                "minItems": 3,
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "pointLongitude": {
+                      "description": "Longitudinal dimension of point.",
+                      "type": "number"
+                    },
+                    "pointLatitude": {
+                      "description": "Latitudinal dimension of point.",
+                      "type": "number"
+                    }
+                  },
+                  "additionalProperties": false,
+                  "required": [
+                    "pointLongitude",
+                    "pointLatitude"
+                  ]
+                }
+              }
+            }
+          },
+          "geoLocationPolygons": {
+            "description": "A drawn polygon area, defined by a set of points and lines connecting the points in a closed chain.",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "polygonPoints": {
+                  "description": "A point location in a polygon.",
+                  "type": "array",
+                  "minItems": 3,
```

Comment:
There's a confusion with this ``minItems: 3``, since DataCite requires 4 (first and last point must match). See point "4" in this list: https://github.com/inveniosoftware/datacite/issues/35 Perhaps we should change it to 4? Alternative is to auto-generate the last point.

Reply:
Yes, let's change to 4 and call it a bug :-)